README.md

```diff
@@ -1,3 +1,17 @@
 Ironic Prometheus Exporter
 ==========================
 
+
+Configuration
+=============
```

Comment:
nit: this should be under the top heading, not on the same level

Reply:
ack